frame/staking/src/benchmarking.rs

```diff
@@ -326,7 +326,7 @@ benchmarks! {
 			n,
 			T::MaxNominatorRewardedPerValidator::get() as u32,
 			false,
-			RewardDestination::Staked,
+			RewardPolicy::One(RewardDestination::Staked),
```

Comment:
The worst case should be paying to one dead account and staked, (or maybe  2 dead account)